src/model/trust.page.model.ts

```diff
@@ -43,28 +44,29 @@ type IndividualTrusteesFormCommon = {
   type: RoleWithinTrustType,
   forename: string,
   surname: string,
-  date_of_birth_day: string,
-  date_of_birth_month: string,
-  date_of_birth_year: string,
+  dateOfBirthDay: string,
```

Comment:
I would keep the field names in the web models as snake case , I don't think we should mix between camel case and snake_case field names in an object/type